src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,202 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> node = new Node<>(value, tail, null);
+        if (head == null) {
+            head = node;
+            tail = head;
+        } else {
+            tail.next = node;
+            tail = node;
```

Comment:
Here use the same logic like in addToTail method. You can use that method here, but don't forget that you increase size inside addToTail method